app/src/NativeBase/Components/Forms/DateTime.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file DatePicker.
+ */
+
+import React, { FC } from 'react'
+import DatePicker from 'react-native-date-picker'
+import { VStack, Box, Flex, Text } from 'native-base'
+
+interface DateTimeProps {
+  description: string
+  maximumDate?: Date
+  minimumDate?: Date
+  mode: 'date' | 'datetime' | 'time'
+  onChange: (date: Date) => void
+  value: Date
+  color: string
+}
+
+const DateTime: FC<DateTimeProps> = ({
+  description,
+  maximumDate,
+  minimumDate,
+  mode,
+  onChange,
+  value,
+  color,
+}) => {
+  return (
+    <VStack>
+      <Box marginTop="24px" paddingLeft="24px">
+        <Text
+          fontSize="12px"
+          margin="0px"
+          lineHeight="7px"
+          color="#939799"
+          paddingY="0px"
+        >
+          {description}
+        </Text>
+        <Text margin="0px" lineHeight="9px" fontSize="md">
+          {value.toLocaleString('en-GB', {
```

Comment:
If this is just to display a date here, you can use `dayjs` which is [pretty versatile](https://day.js.org/docs/en/display/format) and we already have it installed in the app.  There might not be any big problem making that change in `app/android/app/build.gradle`, but generally better not to make the app a larger download than it needs to be, so if we could use `dayjs` here you could avoid that.